templates/data/variable_info.html

```diff
@@ -4,8 +4,8 @@
   </div>
   <div class="panel-body">
     <p>
-      <b>Variable name:</b>
-      <a href="{{ variable }}">{{ variable.name }}</a>
+      <b>Variable name (case sensitiv):</b>
```

Comment:
As this is facing the user, please use "case sensitive".

Reply:
Done.